tailscale/rootfs/etc/s6-overlay/s6-rc.d/tailscaled/run

```diff
@@ -9,10 +9,16 @@ declare -a options
 bashio::log.info 'Starting Tailscale...'
 
 options+=(--state=/data/tailscaled.state)
+# Opt out of client log upload
+options+=(--no-logs-no-support)
 options+=(--tun=userspace-networking)
 
 # Trigger post process to run, after stuff got connected
 ./post &
 
 # Run Tailscale
-exec /opt/tailscaled "${options[@]}"
+if bashio::debug ; then
+    exec /opt/tailscaled "${options[@]}"
+else
+    exec /opt/tailscaled "${options[@]}" &> /dev/null
```

Comment:
I'm not a fan of this, mostly because it isn't controlling log levels; it is just suppressing all output.

The problem with that is that a user ends up with 'nothing', making them wonder if it works even (we get lots of complains with other add-ons that actually use software that don't output anything by nature).

The tailscale project has been reducing logs output quite a lot over time. As for writing logs, I don't consider that a concern tbh; if we can reduce the log level, sure! I'll be all for it, but that isn't the case here, as we already run on log level 0 (the default).

../Frenck